src/main.py

```
@@ -1943,14 +2631,30 @@ def OnChar(self, event):
                             text_area.AutoCompShow(len(current_word), completions)
 
                         self.OnSave(wx.EVT_CHAR)
-                        self.SetStatusText("    Autosaved", 2)
             except Exception as e:
-                self.SetStatusText(f"    Error running onchar: {str(e)}", 2)
+                print("well shit its broken again. you found the gem in the code")
 
         event.Skip()  # Continue processing other key events
+
+    def on_activate(self, event):
```

Ruff (0.11.9)

Redefinition of unused `on_activate` from line 260

(F811)